docs/UserGuide.md

```diff
@@ -120,19 +123,24 @@ Examples:
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 role/supplier`
```

Comment:
I personally feel its ok to remove the general examples, and just have the 3 diff roles with their examples to reduce clutter

Reply:
agree ill remove